libs/blocks/global-navigation/global-navigation.js

```diff
@@ -214,24 +231,7 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getUniversalNavLocale = (locale) => {
-  const LANGMAP = {
-    cs: ['cz'],
-    da: ['dk'],
-    de: ['at'],
-    en: ['africa', 'au', 'ca', 'ie', 'in', 'mt', 'ng', 'nz', 'sg', 'za'],
-    es: ['ar', 'cl', 'co', 'cr', 'ec', 'gt', 'la', 'mx', 'pe', 'pr'],
-    et: ['ee'],
-    ja: ['jp'],
-    ko: ['kr'],
-    nb: ['no'],
-    pt: ['br'],
-    sl: ['si'],
-    sv: ['se'],
-    uk: ['ua'],
-    zh: ['cn', 'tw'],
-  };
-
+export const getUniversalNavLocale = (locale) => {
```

Comment:
Your suggested approach was my initial way of doing it, but after discussions with Narcis, we decided to export the `getUniversalNavLocale` and test directly on it, because initialising the gnav for every locale was exceeding the timeout limit for the tests.